adding ValidationsProfile swagger

purbch opened the pull request:
## ARM API Information (Control Plane)

<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow.
</i>
Azure 1st Party Service can try out the [Shift Left](https://aka.ms/ShiftLeft) experience to initiate API design review from ADO code repo.  If you are interested, may request engineering support by filling in with the form https://aka.ms/ShiftLeftSupportForm.

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [X] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month. Feature is already deployed
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month. By this month
  4. By default, Azure SDKs of all languages (.NET/Python/Java/JavaScript for both management-plane SDK and data-plane SDK, Go for management-plane SDK only ) MUST be refreshed with/after swagger of new version is published. If you prefer NOT to refresh any specific SDK language upon swagger updates in the current PR, please leave details with justification here.

### Contribution checklist (MS Employees Only):
- [X] I commit to follow the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy) of "no breaking changes"
- [X] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist

> **Applicability**: :warning:
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs
> - Adding new properties
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [X] Check this box if any of the following apply to the PR so that the label "ARMReview" and "WaitForARMFeedback" will be added by bot to kick off ARM API Review.  Missing to check this box in the following scenario may result in delays to the ARM manifest review and deployment.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you copy the existing version into the new directory structure for first commit and then push new changes, including version updates, in separate commits. You can use OpenAPIHub to initialize the PR for adding a new version. For more details refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).

- [ ] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them.

### Breaking Change Review Checklist
If you have any breaking changes as defined in the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy/), request approval from the Breaking Change Review Board.

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Additional details on the process and office hours are on the [Breaking Change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

NOTE: To update API(s) in public preview for over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

himasekhar-ms committed:
Himasekhar ms mediaservices microsoft.media 2023 01 01 (#22557)

* Adds base for updating Microsoft.Media/Metadata from version stable/2022-08-01 to version 2023-01-01

* Updates readme

* Updates API version in new specs and examples

* Adding new version 2023-01-01 for metadata APIs which includes ecryption scope support in Assets create API

* Updating AssetsAndAssetFilters manifest with new property encryptionScope and adding new test case for asset create with encryption scope

* Fixed text alignment issues in the example file

* Adding new example file in AssetsAndAssetFilters

* Add missing swagger files and update tag name

* Fixing the path for the Swagger file

* Updating stemingService version to 2022-01-01

* Updating stemingService version to 2022-08-01

* Reverting the StremingServive API version changes

* fix the test readMe file

* fix the test readMe input file version

---------

Co-authored-by: giakas <giakas@microsoft.com>

jha1bhavna committed:
Adding None as allowed values for TLS (#22949)

chunyu3 committed:
upgrade AnomalyDetector to use typespec library 0.41.0 (#23036)

* upgrade AnomalyDetector to use typespec library 0.41

* updated package.json

* remove un-used python emitter dependency

chunyu3 committed:
upgrade openapi to typespec library 0.41.0 (#23038)

* upgrade openapi to tyepspec library 0.41.0

* add typespec as custom word

* update package.json

TimLovellSmith committed:
Refactor cache swagger to deduplicate SkuName enum definitions. (#23049)

anjaliOra committed:
[Hub Generated] Publish private branch 'datafactory/anjshar/adfNewSwaggerUpdateForIR' (#23122)

* [AutoSync] ecc97ae10f new process private swagger update

* [AutoSync] 45ff9fdbbb moving additionalProps out of props

* [AutoSync] 0287001cba adding identifier

---------

Co-authored-by: swagger-automation <swagger@microsoft.com>

giakas committed:
Adding a custom tag with older streaming for C#SDK (#23126)

ofermoses committed:
removing mcas from examples in MDC settings api docs as intergration is being deprecated. (#23072)

no changes to actual API client, just docs

Co-authored-by: mosesofer@microsoft.com <mosesofer@microsoft.com>

bexxx committed:
Add support for ChatGPT (chats/completion) for Azure OpenAI (#23074)

* Added old spec under new location as base

* Add chat completion API to inference specs

* Add custom word

* Fixed model name and product name

bexxx committed:
Add support for ChatGPT to Azure OpenAI authoring APIs (#23076)

* Add old api version as base for new

* Added changes for new version

* Fix reference in readme

croos-msft committed:
Add Supporting Windows Docker Container Hosted API Services (#22587)

* Add windowsdatacontainer and pipeline changes

* Add pipeline changes

* Fix hardcoded brnch

* Copy git file to whichever Dockerfile path is being used

* Remove changes to data-container pipeline

* Setup windows pipeline

* Update with correct ps1 location

* Fix issues with dockerfile

* Use Windows-styled folders

* Add logic to replace base image

* Add base_image_windows

* Correctly refer to Dockerfile

* Enclose in quotation marks

* Only use servercore:ltsc2022

senatol committed:
Update operation ids (#23100)

tfitzmac committed:
clarify when reserved is required (#22860)

Alancere committed:
[Go] newrelic config readme.go.md (#23148)

minnieliu committed:
Feature/room api2023 (#22540)

* Modify file for the new patch participants API

* Participant endpoint changes for Rooms Api

Introduce JMP for particpant collection.

* Fix directory structure

* Copy examples from previous version

* update examples

* Update swagger json and examples

* Updated ParticipantsCollection property to use value instead participant and modified its descriptions

* Update rooms json examples according to new changes (#22495)

* Update rooms json examples according to new changes

* Address pipeline validation errors

* Address pipeline validation errors

* Address pipeline validation errors

* revert readme

---------

Co-authored-by: Minnie Liu <peiliu@microsoft.com>

* Update participant collection (#22498)

* Updated Participant Collection Property  to use value in place of participant. Updated API sample

* Updated Rooms_GetParticipants sample

* Ran Prettier

* [Rooms] Removed roomJoinPolicy and renamed RoleType to Role (#22494)

* Added Title Property to RoomModel, CreateRoomRequest, UpdateRoomRequest

* Revert "Added Title Property to RoomModel, CreateRoomRequest, UpdateRoomRequest"

This reverts commit 48e46b0eac97bc5c29f7b782ddeb9c869948499c.

* Added title property to RoomModel, CreateRoomRequest, UpdateRoomRequest (#22513)

* Revert all changes in the 2022-02-01 swagger (#22539)

Co-authored-by: Minnie Liu <peiliu@microsoft.com>

* Revert all changes in the 2022-02-01 swagger

* Add required request body for patch room (#22589)

* Add required request body for patch room

* Update update room example

* Fix validation checks

---------

Co-authored-by: Minnie Liu <peiliu@microsoft.com>

* Update authorization description and remove format in Endpoint (#22609)

* Update authorization description

* Remove format in endpoint to align with other ACS services

---------

Co-authored-by: Minnie Liu <peiliu@microsoft.com>

* Updated Rooms API Specification Per Internal Review (#22608)

* Updated Room API Spec per internal review

* Fixed Validation Check Failures

* Fixed Validation Failures on role type

* Updated default values descriptions

* Updated Readme with new API version

* Clean up and fix communication error in spec (#22621)

* Clean up and fix communication error in spec

* Fix example version

---------

Co-authored-by: Minnie Liu <peiliu@microsoft.com>

* Removed Room Title Per Internal Review (#22640)

* Add a new participant model with role as optional (#22641)

Co-authored-by: Minnie Liu <peiliu@microsoft.com>

* createdAt name change (#22651)

* Remove Rooms 2023-01-01 version to clean up (#22676)

Co-authored-by: Minnie Liu <peiliu@microsoft.com>

* Remove bad examples (#22682)

Co-authored-by: Minnie Liu <peiliu@microsoft.com>

* Update current feature branch with the changes from DevEx review (#22859)

* Update current feature branch with the changes from DevEx review

* Update samples

* Merge patch samples

* fix

* Missed comment from devex review

* Missed comment from devex review

* fix grammar

* fix grammar

---------

Co-authored-by: Minnie Liu <peiliu@microsoft.com>

* Sort models in alphabetical order (#22886)

Co-authored-by: Minnie Liu <peiliu@microsoft.com>

* Add new get rooms list API (#22936)

* Update format of repeatability-first-sent (#22968)

* Update format of repeatability-first-sent

* Update in correct version

---------

Co-authored-by: Minnie Liu <peiliu@microsoft.com>

* Update readme file to include previous version (#23002)

* Update readme file to include previous version

* Fix room ids

* nit changes to list rooms API

---------

Co-authored-by: Minnie Liu <peiliu@microsoft.com>

* Update API Specification per preparation for ARB

* Fix nit

* Renamed Particpants_Patch to Participants_Update

* Fix validation errors

* Create Room should take a map of participants (#23090)

* Create Room should take a map of participants

* Fix Room Id example

---------

Co-authored-by: Minnie Liu <peiliu@microsoft.com>

* Fix validFrom and validUntil exampls

* Add period to descriptions

* Update description

* Update description for create room participants

---------

Co-authored-by: HuangXin-MS <94944762+HuangXin-MS@users.noreply.github.com>
Co-authored-by: anujissarMS <anujissar@microsoft.com>
Co-authored-by: Mike Hang <108955604+mikehang-msft@users.noreply.github.com>
Co-authored-by: Minnie Liu <peiliu@microsoft.com>
Co-authored-by: Mrayyan <t-siddiquim@microsoft.com>

deyaaeldeen committed:
[Language Text] Delete Dynamic Classification from v2022-10-01-preview (#23080)

* [Language Text] Delete Dynamic Classification from v2022-10-01-preview

* remove from dev

matthohn-msft committed:
[Hub Generated] Review request for Microsoft.Communication to add version preview/2023-03-01-preview (#22996)

* Adds base for updating Microsoft.Communication from version preview/2021-10-01-preview to version 2023-03-01-preview

* Updates readme

* Updates API version in new specs and examples

* Adding version 2023-03-01-preview changes.

* Adding missing readme.md update

* Addressing PR comments.

* Updating to v5 common types

* Incorporating comments from the stable version PR

* Updating description for the provisioning state enum

* Fixing model validation errors

* Removing redundant systemData property.

* Adding 201 response to indicate resource was create successfully

dolauli committed:
Enable Azure PowerShell CI for liftrqumulo (#22862)

chcmsft committed:
update description of geocoding api (#23101)

guptashweta9 committed:
G3 G4 SKU removal (#23099)

rahuls-microsoft committed:
Updating example for Linter Errors. (#23174)

Co-authored-by: Rahul Singh <rahul.station99@gmail.com>

prchalla1 committed:
To add Update Cost Details Field Link in GenerateCostDetailsReport (#23137)

jorgecotillo committed:
OperationalInsights fix missing enum value (#23123)

ismael-ms committed:
Fixes for Features APIs' descriptions (#23001)

* Remove ontology from API descriptions and clarify that properties are defined in the collection definition API

* Specify GET Collection Definition

* Add missing descriptions

* Add more missing descriptions

* Change 'Response' to 'Result'

* Fix casing of descriptions

swathipil committed:
add data-plane ServiceBus swagger to main (#23057)

* add back sb swagger

* update api version param defintion

* update api version num

* prettier

* spellcheck/readme

* readme/lint

* suppress x-ms-examples not found err

purbch committed:
adding ValidationsProfile swagger

charmainewkchan committed:
Make failover group status type object to fix resource sync (#23058)

* Make status object to fix resource sync

* Remove trailing comma

* Move changes to 2023-03-15-preview

---------

Co-authored-by: Charmaine Chan <chachan@microsoft.com>

Alancere committed:
[Go] paloaltonnetworks readme.go.md config (#23187)

FumingZhang committed:
Update custom words (#23034)

shanhix1 committed:
Edit description of properties.provisioningState (#23146)

bojunehsu committed:
Add 2023-02-28-preview. (#23093)

* Add 2023-02-28-preview.

* Update cSpell.json

purbch committed:
adding the lintDiff

purbch committed:
fixing the semantic diff

allison-inman committed:
[Hub Generated] Review request for Microsoft.Quota to add version stable/2023-02-01 (#22367)

* Adds base for updating Microsoft.Quota from version preview/2021-03-15-preview to version 2023-02-01

* Updates readme

* Updates API version in new specs and examples

* Fix version in readme

* Remove stale version, update readme

* Mark parameter as optional

asaflevi-ms committed:
[Cognitive Services - Health Insights] CADL revision for public preview (#22990)

* health insights cadl + generated openapi spec

* fix PR Comments

* change country to countryOrRegion

* split to multiple cadl projects

* update cadl-project.yaml files

* update example with generic contact details

* update samples with generic contacts

* convert cadl to typespec

* update generated openapi spec

* convert cadl to typespec (#4)

* convert cadl to typespec

* update generated openapi spec

* move ClinicalNoteEvidence to common modules

* remove redundant imports

* fix typespec dependencies

* remove dependencies from main.tsp

* using rest

* move imports from service.tsp to its caller

* fix comment - directive error

* remove imports from service.tsp

iscai-msft committed:
make path and body positional (#23185)

MichaelMSWork committed:
[Hub Generated] Review request for Microsoft.SaaS to add version preview/2018-03-01-beta (#22911)

approved with false breaking change

BharathaAravind committed:
[Hub Generated] Review request for Microsoft.Help to add version preview/2023-01-01-preview (#23056)

* Generate new Service for help

* init commit microsoft.help-2023-01-01-preview

* Updating swagger documentation #1

* Updating swagger documentation #2

* Adding suppressions for Help 2023-01-01-preview api

* Adding suppressions for Help 2023-01-01-preview api #2

* removed unnecessary files

* update examples

AntonioHernandez16 committed:
[Hub Generated] Review request for Microsoft.Consumption to add version stable/2022-09-01 (#23054)

* Adds base for updating Microsoft.Consumption from version stable/2021-10-01 to version 2022-09-01

* Updates readme

* Updates API version in new specs and examples

* adding new API version (September Copper) to Swagger

* fixing prettier check

FumingZhang committed:
Review request for Microsoft.ContainerService/aks to add version 2023-02-01 & 2023-02-02preview (#23189)

* release 2023-02-01 & 2023-02-02-preview

* patch changes to previous API versions

jiayi11 committed:
Microsoft.AppConfiguration: Add new 2023-03-01 version (#22788)

* Adds base for updating Microsoft.AppConfiguration from version stable/2022-05-01 to version 2023-03-01

* Updates readme

* Updates API version in new specs and examples

* Add new APIs in 2023-03-01

* Delete annotations.json

* Delete assets.json

* Update readme.test.md

* Update basic.yaml

---------

Co-authored-by: Lei Ni <7233663+leni-msft@users.noreply.github.com>

staladay committed:
[Hub Generated] Review request for Microsoft.HealthcareApis to add version stable/2023-02-28 (#22837)

* Adds base for updating Microsoft.HealthcareApis from version stable/2022-12-01 to version 2023-02-28

* Updates readme

* Updates API version in new specs and examples

* Adding "crossTenantCmkApplicationId" property with a new API version and new examples for data sovereign regions.

* Fixing some issues based on automatic validator feedback.

* Removed extra space causing model validation failures.

* Added Azure App ID validation regex for crossTenantCmkApplicationId string property.

mbashtovaya committed:
NetworkCloud 2022-12-12-preview API corrections (#23086)

* first public preview version for NetworkCloud APIs

* added custom words

* added required attribute to payloads to correct aaz-dev-tool CLI generation

* corrected console patch API payload

* removed invalid child resources list by subscription APIs

jeffmcco-msft committed:
Adjust 202 response of UsageAggregates_List to match current behavior (#23172)

maximrytych-ms committed:
[IC3 ACS - Auth] <stable/2023-08-01> (#22306)

* Add new swagger for idempotent APIs

* Update swagger and examples

* Update readme file

* Update header description and fix date format in examples

* Fix spacing in header comments

* Add dummy diff for internal review

* Fix example comment

* Revert "Merge branch 'feature/maximrytych/add-new-swagger-for-idempotent-apis' of https://github.com/maximrytych-ms/azure-rest-api-specs into feature/maximrytych/add-dummy-diff-for-internal-review"

This reverts commit b8b529b7c39e8e23a7239ce8f8ee3dc8ceeaaf49, reversing
changes made to 8aeb21992e34f7a3265455b529ff6c7623683b05.

* Update swagger, samples and readme

* Revert change done by mistake

* Add type for Repeatability-Result

* Update newest package tag in readme

* Fix format of Repeatability-First-Sent header

* Update swagger based on Azure Board review

ashwinisingh01 committed:
[Cosmos DB] Fix CosmosDBMongo Source Sink (#23191)

* fix CosmosMongoSourceLink

* address review comments

* addresss review comments

Alancere committed:
[Go] storagecache add directive (#23218)

* [Go] storagecache add directive

* fix

Wzb123456789 committed:
Update readme.python.md (#23240)

msyyc committed:
Update readme.python.md (#23214)

Alancere committed:
[Go] storagecache add directive (#23242)

apmehrotra committed:
Adding support for child service configuration resource (#23116)

* Adds base for updating Microsoft.HybridConnectivity from version preview/2022-05-01-preview to version 2023-03-15

* Updates readme

* Updates API version in new specs and examples

* Adding support for child service configuration resource

* Addressing PR comments

* Handling camel casing

* Updating the examples

* Addresing issues

Wzb123456789 committed:
Update readme.python.md (#23239)

* Update readme.python.md

* Update readme.python.md

* Update readme.python.md

---------

Co-authored-by: Yuchao Yan <yuchaoyan@microsoft.com>

bojunehsu committed:
Update owners for Form Recognizer (#23241)

zmssp committed:
[AutoSync] a2b6e0791 Merged PR 7757123: fix: fix create build service api (#23213)

Co-authored-by: swagger-automation <swagger@microsoft.com>

Ameer-az committed:
[Hub Generated] Review request for Microsoft.Security to add version preview/2023-03-01-preview (#23227)

* Adds base for updating Microsoft.Security from version preview/2022-08-01-preview to version 2023-03-01-preview

* Updates readme

* Updates API version in new specs and examples

* 03 version changes

* change readme tag

* update composite v3

---------

Co-authored-by: Ameer Abu Zhaia <amabuzha@microsoft.com>

moadus committed:
add or modify files (#23193)

meydand committed:
[Hub Generated] Review request for Microsoft.AlertsManagement to add version preview/2023-04-01-preview (#22849)

* Adds base for updating Microsoft.AlertsManagement from version preview/2023-01-01-preview to version 2023-04-01-preview

* Updates readme

* Updates API version in new specs and examples

* Add TenantActivityLogAlerts Resource type

* Prettier

* Align with ALA

* Fix issues

* Prettier

* Fix param

* fix PR comments

---------

Co-authored-by: meydand <t-meday@microsoft.com>
Co-authored-by: Meydan Dayan <meday@microsoft.com>

bjqian committed:
Azure SignalR - Adding new api-version 2023-03-01-preview (#22948)

* init 2023-03-01-preview for signalr and webpubsub. Copied from 2022-02-01

* add new api version 2023-03-01-preview

* revert changes to package-lock.json and package.json

* 1. Add restart to replica
2. replica tracedResource, subscriptionId, resourceGroup, apiVersion within commonTYpe in v5
3. return 202 for patch operation
4. add 'x-ms-long-running-operation-options' for put operation
5. Add pattern to resourceName and replicaName

* 1. Change response status code to 202 for patch
2. Fix the example naming
3. Add location header for post 202, change  'final-state-via' property to 'location' for POST

* 1. Fix example response body
2. Fix "x-ms-long-running-operation-options" for put

* 1. Fix example response body
2. Fix LocationHeader for PATCH

* 1. Remove SystemData form models inherited from "Resource"
2. Reuse trackedResource, privateEndpointConnection, privateEndpointConnectionName

* Fix default value of clientCertEnabled

* Revert privateEndpointConnection since model in common is different from ours

* Fix the resourceName pattern and example

* fix naming pattern

* 1. Add 200 for signalrRegenerate Key
2. fmt example

raych1 committed:
Updated pr assignment config (#23258)

raosuhas commented on specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2022-08-03/gallery.json:
```diff
@@ -2760,6 +2795,29 @@
+        },
+        "status": {
+          "type": "string",
+          "description": "This property specifies the status of the validationProfile of the image version."
```
consider updating to enums if applicable

purbch replied:
which enum? do you mean to add in the status?

raosuhas commented:
Signing off on this with a minor comment. Your PR does have a large number of Model errors though and those will need to be fixed before checking from the feature branch into main FYI : 

https://github.com/Azure/azure-rest-api-specs/pull/23201/checks?check_run_id=12174883691

vidit-msft committed:
update Elastic Pool version from 2021-08-01-preview to 2022-08-01-preview (#23232)

purbch committed:
fixing the prettier-check

raych1 committed:
TypeSpec renaming for specificationRepositoryConfiguration (#23273)

chunyu3 committed:
update configure (#23245)

Wzb123456789 committed:
Update readme.python.md (#22690)

* Update readme.python.md

* Update readme.python.md

* Update readme.python.md

---------

Co-authored-by: Yuchao Yan <yuchaoyan@microsoft.com>

Wzb123456789 committed:
Update readme.python.md (#23274)

yanfa317 committed:
Merge branch Dev release network 2022 06 01 preview into main branch (#23228)

* Adds base for updating Microsoft.Network from version preview/2022-04-01-preview to version 2022-06-01-preview

* Updates readme

* Updates API version in new specs and examples

* Add New ListNetworkGroupMembership API to  network 2022 06 01 preview (#21970)

* New networkManagerGroupMembership API

* fix typo

* New List NetworkGroupMembership API

* fix

* fix format

* fix error

* fix

* fix as comments

* fix

priyjain358 committed:
[Hub Generated] Review request for Microsoft.AzureStackHCI to add version stable/2023-03-01 (#23255)

* Adds base for updating Microsoft.AzureStackHCI from version stable/2023-02-01 to version 2023-03-01

* Updates readme

* Updates API version in new specs and examples

* Consent and Disable APIs added

* add managed by for extensions

* prettier fix

thramu committed:
[Hub Generated] Review request for Microsoft.Consumption to add version stable/2023-03-01 (#23264)

* Adds base for updating Microsoft.Consumption from version stable/2022-09-01 to version 2023-03-01

* Updates readme

* Updates API version in new specs and examples

* Adding new API version contract changes to 2023-03-01 (#22644)

* Added RI Tranastions query parameters to billing account scope. Updat… (#22963)

* Added RI Tranastions query parameters to billing account scope. Updated example. Added documentation around size response limit

* Ran prettify and fixed typo

* Adding suppression for valid formats

* Single directives (#23014)

---------

Co-authored-by: MattEberhart-MSFTE <107518095+MattEberhart-MSFTE@users.noreply.github.com>

swathipil committed:
update servicebus swagger required/types (#23225)

* update types for title names and required entity descriptions

* make namespace title required

* add default rule description to subscription description

HarshaNalluru committed:
Update appconfiguration.json to include the "operations" path (#23125)

* Update appconfiguration.json

* Update specification/appconfiguration/data-plane/Microsoft.AppConfiguration/preview/2022-11-01-preview/appconfiguration.json

* Update specification/appconfiguration/data-plane/Microsoft.AppConfiguration/preview/2022-11-01-preview/appconfiguration.json

Co-authored-by: Deyaaeldeen Almahallawi <dealmaha@microsoft.com>

* Update specification/appconfiguration/data-plane/Microsoft.AppConfiguration/preview/2022-11-01-preview/appconfiguration.json

* Update appconfiguration.json - Deya feedback

* example GetOperationStatus

* Update specification/appconfiguration/data-plane/Microsoft.AppConfiguration/preview/2022-11-01-preview/examples/GetOperationStatus.json

Co-authored-by: Deyaaeldeen Almahallawi <dealmaha@microsoft.com>

* Updates to descriptions, names, id format.

* Update operation status to use error detail as defined by Azure REST API guidelines.

* Prettier fixes.

* dummy commit for CI

---------

Co-authored-by: Deyaaeldeen Almahallawi <dealmaha@microsoft.com>
Co-authored-by: Jimmy Campbell <jimmyca@microsoft.com>

purbch committed:
fixing the swagger version

matthohn-msft committed:
[Hub Generated] Review request for Microsoft.Communication to add version stable/2023-03-31 (#23202)

* Adds base for updating Microsoft.Communication from version preview/2023-03-01-preview to version 2023-03-31

* Updates readme

* Updates API version in new specs and examples

msyyc committed:
Update readme.python.md (#23300)

kerend committed:
[Hub Generated] Publish private branch 'simanor-dev-security-Microsoft.Security-2023-01-01' (#23305)

* Pricing new version 2023-01-01

* Rename additionalProperty

* change to AgentlessDiscoveryForKubernetes

* code revirew

* Fix lint error

* Rename Storage extensions

* Remove the 207 response code + add errorMessage

* Adding operationStatus

* Add previous API version files

* Override previous API version files

* Adding PUT partial success example

* DataSensitivityDiscovery to SensitiveDataDiscovery

* Fix PR comments

* PR comments fix

* Fix PR comments

* EnablementTime

* .

* .

* .

* .

---------

Co-authored-by: Sivan Manor <simanor@microsoft.com>
Co-authored-by: Reem Joron <rejoron@microsoft.com>

ismael-ms committed:
Deprecate Maps Elevation API (#23290)

* Add deprecation banner to Maps Elevation

* Add hyperlink for retirement page

ismael-ms committed:
Add deprecation banner to GetMapTile and GetMapTileset (#23307)

LuChen-Microsoft committed:
[ACS - Chat] Data Retention Policy 2023-07-01-preview (#22858)

* Adds base for updating Chat from version stable/2021-09-07 to version 2023-07-01-preview

* Updates readme

* Updates API version in new specs and examples

* add retention policy

* add examples change

* update readme

* fix stable version

* fix validations

* merge the changes with image sharing

* remove unused example

* - Remove the retention policy from updat chat thread
- Rename the `BaseOnThreadCreationDateRetentionPolicy` to `ThreadCreationDateRetentionPolicy`

dk1729 committed:
Removed all examples and specifications for api version 2021-04-04-pr… (#22413)

* Removed all examples and specifications for api version 2021-04-04-preview

* Removed all 2021 api version references from README.md

---------

Co-authored-by: Dhruwaksh Dave <dhrdave@microsoft.com>

raych1 committed:
Update emitter name in sdk auto config (#23312)

paramsiva1 committed:
Add ManagedDatabases Api to Composite V5 (#23302)

Co-authored-by: Parameswara Reddy Bodeddula <pbodeddula@microsoft.com>

profDisarray committed:
[Hub Generated] Review request for Microsoft.RecoveryServices to add version stable/2023-01-15 (#22431)

* Adds base for updating Microsoft.RecoveryServices from version stable/2021-11-15 to version 2023-01-15

* Updates readme

* Updates API version in new specs and examples

* Adds RecoveryPointProperties for 2023-01-15

* Adds RecoveryPointProperties at missed places

* Fixes API version in examples

* Fixes model validation errors

kerend committed:
[Hub Generated] Publish private branch 'security/dahandaniel/seurityOperators' (#23323)

* Add autoAdd field to indicate sku should be included during creation (#5849)

* copy folder

* Add autoAdd field

* update readme

* Add operation example

* fix style

Co-authored-by: Peter Shen <yinshen@microsoft.com>

* Add previour Private PR for 2021-10-15-preview as a baseline

* Update EventGrid.json with new features to be part of 2021-10-15-preview API

* update readme file and add missing example files

* fix typo

* fix operation name for unauthorize

* fix tag

* New API version to Microsoft.Security 2023-01-01-preview

* refactoring

* Update readme

* fix syntax error

* Refactoring

* Add pattern restriction

* Fix readme

* npm run prettier

* Update naming in the examples

* fix merge

* merge fix

* fix merge

* fix merge

* Update readme.md

* Update readme.md

---------

Co-authored-by: Peter Shen <shen.yingjun@hotmail.com>
Co-authored-by: Peter Shen <yinshen@microsoft.com>
Co-authored-by: Ashraf Hamad <ahamad@ntdev.microsoft.com>
Co-authored-by: Daniel Dahan <dahandaniel@microsoft.com>
Co-authored-by: Sivan Manor <simanor@microsoft.com>

Commit:
Release Microsoft.App 2022-11-01-preview  (#22896)

masenol committed:
[Quantum] Data Plane 2022-09-12-preview (#23183)

* Adding the previous version, 2021-11-01-preview, as the initial commit under 2022-09-12-preview folder for easier review

* Updating the 2022-09-12-preview version

* Removing jobCount from sessionPut example

conniey committed:
Change property from int to long. (#23286)

conniey committed:
Changing modelAsString = true (#23333)

MicroJamesLim committed:
[Hub Generated] Review request for Microsoft.ContainerInstance to add version preview/2023-02-01-preview (#23221)

* Adds base for updating Microsoft.ContainerInstance from version stable/2022-09-01 to version 2023-02-01-preview

* Updates readme

* Updates API version in new specs and examples

* added property

* added sample request and responses

* integer format

* read only property

* read only fix

* custom word

* changed boolean to enum

* changed boolean to enum in examples

* x-ms-enum

fnuarnav committed:
added properties property for operation object (#22846)

* added properties property for operation object

* updated older api version as well

* prettier fix

* use full description for properties object

* remove 'type'

* use correct property name

matthohn-msft committed:
Reverting change in name pattern regex (#23318)

utsrkr committed:
Add error and intervention actions. (#23301)

Add error and intervention actions.

nickdepinet committed:
[Hub Generated] Publish private branch 'nickdepinet-devcenter-Microsoft.DevCenter-2023-01-01-preview' (#23124)

* add or modify files

* Update readme.md

re-adding azure-sdk-for-go

kazrael2119 committed:
update newrelic typescript.md (#23246)

kazrael2119 committed:
update typescript.readme in paloa... (#23297)

larikaS committed:
S360 fix EdgeOrder 2022-05-01-preview (#23293)

* S360 fix EdgeOrder 2022-05-01-preview

* reverting change for referrer

raych1 committed:
Pr assignment change (#23349)

anthony-c-martin committed:
[FabricBot] Update mentions for Azure Deployments team (#23354)

minnieliu committed:
Add pageable support for list functions (#23308)

* Add pageable support for list functions

* nit

---------

Co-authored-by: Minnie Liu <peiliu@microsoft.com>

BharathaAravind committed:
[Hub Generated] Review request for Microsoft.Help to update swagger documentation for version preview/2023-01-01-preview (#23357)

* updating api spec documentation

* updating api spec documentation #2

* updating api spec documentation #3

yirsu committed:
Adding "healthStatus" property to Enterprise Policy model, and fix Private Endpoint Connection DELETE/PUT responses (#23205)

* Add EP healthStatus property

* Fix Private Endpoint swagger issue

* Remove 202 response from Private Endpoint Connection model

prchalla1 committed:
[Hub Generated] Review request for Microsoft.CostManagement to add version stable/2022-10-01 (#23262)

merged with breaking change approved

abhayohri committed:
[Hub Generated] Review request for Microsoft.ResourceHealth to add version stable/2022-05-01 (#22916)

* make fixes for s360 items

* spelling fix

* added missing properties to all stable api versions

* fix model validation errors

t-bzhan committed:
[CDN] Add missing default values to fix swagger quality issues in s360 (#23029)

* Add missing default value to fix s360

* Fix PrettierCheck issue

---------

Co-authored-by: Bo Zhang <bzhan@microsoft.com>

bhargav-kansagara committed:
[Agrifood] Adding new API version 2023-04-01-preview (#23265)

* Adding 2022-11-01-preview swagger for reference

* Adding provicer client id and client secret as part of the Scene Create Job and Scenes Get & Search Features APIs

* Removing duplicate parameter IntegrationIds from list sensor integrations API

* Add changes for supporting multi-polygon shapefile ingestion

---------

Co-authored-by: Bhargav Kansagara <bhkansag@microsoft.com>
Co-authored-by: Pratyush Prateek <pratyush.prateek.eee15@itbhu.ac.in>

ShawnJackson committed:
Update custom-words.txt (#23379)

guy-microsoft committed:
Healthbot sdk fix - add "x-ms-long-running-operation": true to patch operation (#22908)

* Update healthbot.json

Add "x-ms-long-running-operation": true

* Update healthbot.json

Add "x-ms-long-running-operation": true

* Update healthbot.json

Add "x-ms-long-running-operation": true

* Update healthbot.json

Add "x-ms-long-running-operation": true

* Update healthbot.json

* Update healthbot.json

Fix

* Update healthbot.json

* Update healthbot.json

* Update healthbot.json

Sandido committed:
Update Image aliases with new versioned aliases  (#23261)

* Update aliases.json

* remove 2008 alias

this image was entirely deprecated, and was removed from PS.

* add centos versioned

* add architecture

* Update arm-compute/quickstart-templates/aliases.json

Co-authored-by: wu5bocheng <wu5bocheng@gmail.com>

* Update arm-compute/quickstart-templates/aliases.json

Co-authored-by: wu5bocheng <wu5bocheng@gmail.com>

* Update arm-compute/quickstart-templates/aliases.json

Co-authored-by: Xing Zhou <Zhou.Xing@microsoft.com>

* Update arm-compute/quickstart-templates/aliases.json

Co-authored-by: wu5bocheng <wu5bocheng@gmail.com>

* Update arm-compute/quickstart-templates/aliases.json

Co-authored-by: Xing Zhou <Zhou.Xing@microsoft.com>

* Update arm-compute/quickstart-templates/aliases.json

Co-authored-by: wu5bocheng <wu5bocheng@gmail.com>

* Update arm-compute/quickstart-templates/aliases.json

Co-authored-by: Xing Zhou <Zhou.Xing@microsoft.com>

* Update arm-compute/quickstart-templates/aliases.json

Co-authored-by: Xing Zhou <Zhou.Xing@microsoft.com>

* Update aliases.json

---------

Co-authored-by: wu5bocheng <wu5bocheng@gmail.com>
Co-authored-by: Xing Zhou <Zhou.Xing@microsoft.com>

msyyc committed:
Update readme.python.md (#23296)

* Update readme.python.md

* Update readme.python.md

* Update readme.python.md

chrissmiller committed:
Publish Microsoft.DevCenter 2023-04-01 stable API version (#22970)

* Add base

* Update API version

* Update readme

abhayohri committed:
[Hub Generated] Review request for Microsoft.ResourceHealth to add version stable/2022-10-01 (#22721)

* Adds base for updating Microsoft.ResourceHealth from version preview/2022-10-01-preview to version 2022-10-01

* Updates readme

* Updates API version in new specs and examples

* added missing api

* Revert "added missing api"

This reverts commit 111d6ca9130b9591a25bbf3a2643037b5e5846d8.

* added missing api

* added error message suppression to readme

* prettier fix

* added missing fields for s360 items

* updated description

* spelling fix

* added missing properties to all stable api versions

* Revert "added missing properties to all stable api versions"

This reverts commit 220764bd5349cd657e0043d5ccc09dd6dc2e4a3b.

* fixes from pr for resourcehealth.json file

* updated child avialbility example uri

* added suppression for _ActionUrl

* added suppression for _ActionUrl

* added suppression for _ActionUrl

* added suppression for _ActionUrl

* added suppression for _ActionUrl

* added suppression for _ActionUrl

* linter wrnings

* update readme for .net sdk

* update readme for .net sdk

* update readme for sdks

* trying suppression for actionurl.comment

* added suppression for _ActionUrl

ArthurMa1978 committed:
Filter out all Autorization header value to avoid CredScan alarm (#23380)

msyyc committed:
Update readme.python.md (#23381)

ShawnJackson committed:
[AQ] edit pass: REST API reference docs (#23288)

* edit pass: rest-api-reference-docs

* edit pass: rest-api-reference-docs

* edit pass: rest-api-reference-docs

* removing file

* blacklist to blocklist

iscai-msft committed:
add using Azure.Core to mvad (#23384)

LuChen-Microsoft committed:
update swagger version (#23360)

koyasu221b committed:
[Hub Generated] Review request for Route to add version preview/2023-08-01-preview (#23044)

* Adds base for updating Route from version preview/1.0 to version 2023-05-31-preview

* Updates readme

* Updates API version in new specs and examples

* Add directions API

* Update example

* fix lint

* update api-version

* fix lint

* rename to pointTypes

* Add readOnly and reaname object

* Add properties object and add x-ms-client-flatten=true

* Add alternative routes and fix json description

* Update routeOutputOptions

* Add nextLink for batch response

* Add geoJSON and Add 207

* rename details to steps

* Add x-ms-client-name for legs, sublegs and steps

* Add required fields

* Add pathLegRange under legs

* rename Route_PostDirectionsBatch to Route_PostRouteDirectionsBatch

* Fix OperationIdNounVerb

* Fix nextLink

* Remove kind

* two discriminator

* Add description

* Rename JSON object and use enum for featuresProperteisType

---------

Co-authored-by: Will Huang <huangwill@microsoft.com>

truptishet committed:
Changes in shippingAddress and encryptionPreferences as a part of S360 (#23332)

* Changes in get call

* Changes in get call

* Changes in get call

* Changes

* Hardware encryption changes

raych1 committed:
Updated sdk automation docs (#23345)

* Updated sdk automation docs

* Update documentation/sdkautomation/README.md

ambrahma committed:
Fix bug where ReplicationRoles was exposing unsupported roles (#23294)

* Fix bug where ReplicationRoles was exposing unsupported roles

* Remove unsupported ReplicationRole from api version 2022-03-08-preview

XiaofeiCao committed:
remove 'selector' and 'selectorMatchOperator' from ApplicationGatewayFirewallExclusion (#23382)

WendyCaiMS committed:
[Hub Generated] Review request for Microsoft.Devices to add version preview/2023-03-01-preview (#22648)

* Adds base for updating Microsoft.Devices from version stable/2022-12-12 to version 2023-03-01-preview

* Updates readme

* Updates API version in new specs and examples

* added identity property to json and examples

* add DPSUpdate example and update identity values

* added identity to PATCH response

* fixed typo

* change to use common managedidentity type

ZhangYi19941217 committed:
Update IpAllocationDelete.json to fix the Linter Error (#23257)

Update IpAllocationDelete.json to fix the Linter Error ( LRO_RESPONSE_HEADER ) for Microsoft.Network:
https://portal.azure-devex-tools.com/amekpis/linting/detail?errorId=EC9B7719-B296-4C59-A8E7-1F5C49E1E916

msyyc committed:
seal single value enum (#23393)

msyyc committed:
Update readme.python.md (#23394)

Alancere committed:
[Go] add directive (#23395)

* [Go] add directive

* billingbenefits add directive, storageimportexport remove directive

dzeliar committed:
[Hub Generated] Review request for SipRouting to add version preview/2023-04-01-preview (#23374)

* Adds base for updating SipRouting from version stable/2023-03-01 to version 2023-04-01-preview

* Updates readme

* Updates API version in new specs and examples

* add preview functionality on top of GA

lmazuel committed:
Accept "preview" in the description (#23358)

Commit:
MVAD edit pass (#23378)

* Edits to OpenAPI doc

* Reflect AQ edits back to typespec and regenerate openapi.json

* Address PR review comments

MikeyMCZ committed:
Adding Definition for Microsoft Translation service (#23231)

* Adding translator API definition (#1)

* Adding translator API definition

* Fixing PR comments, part 1

* Fixes and python

* Add versions

* fix PR comments

* fix operation definition

* Enable all operations and dotnet

* Fixes for languages API

* Double clients and new auth method

* PR comment fix

* Switch to custom operations

* Removing the APIM client

* Adding emitters

* Update api-version

* Updates to models

* Attempt to fix the api-version

* Fixing API version

* Fix dictionaries.

* Fix the models

* Models fixes

* Removing detect, fix breaksentence

* Adding examples

* Fixing languages endpoint

* Updating generation

---------

Co-authored-by: Michal Materna <mimat@microsoft.com>
Co-authored-by: Catalina Peralta <catalinaperaltah@hotmail.com>

* Fix the packages names

* Update model names

* Move files

* Switch to knowValues

* Reuse transliteration text models

* Migrate to TSP

* Updating TS files to satisfy pipeline

* Adding autorest

* Adding swagger file

* Adding examples

* Fixing path to examples

* Speel check fix

* Updating name of TS exporter

* Updating openapi.json

* Linting, spell check fixes

* Fix spelling

* Test removing generators

* Fixing BS example

* Fixing model generation

* Update examples

* Prettier

* Generate new openapi.json

* adding azure-resource-provider-folder config

* Suppress warning

* Remove sdk-for-net-track2

* Changing the name of the folder as per PR comment

---------

Co-authored-by: Michal Materna <mimat@microsoft.com>
Co-authored-by: Catalina Peralta <catalinaperaltah@hotmail.com>

yjsong11 committed:
Add Azure-AsyncOperation header to async operations (#23410)

* add

* fix

---------

Co-authored-by: Younjin Song <younjinsong@microsoft.com>

RenzoPrettoMS committed:
[Hub Generated] Review request for Microsoft.Chaos to add version preview/2023-04-01-preview (#23171)

* Adds base for updating Microsoft.Chaos from version preview/2022-10-01-preview to version 2023-04-01-preview

* Updates readme

* Updates API version in new specs and examples

* chore: update resource identity type and add user assigned identity type

* chore: added patch call and experiment update definition

* chore: add patch example and fix patch validation issues

* chore: fixed lint and prettier checks

* chore: add permission information for capability types

* chore: add permission information for capability types and fix lint issue

* chore: update example for lint issue

* fixed prettier error

---------

Co-authored-by: Brent Rusinow <brrusino@microsoft.com>

AndresZourelli committed:
[Hub Generated] Review request for Microsoft.AzureActiveDirectory to add version preview/2023-01-18-preview (#22236)

* Adds base for updating Microsoft.AzureActiveDirectory from version stable/2021-04-01 to version 2023-01-18-preview

* Updates readme

* Updates API version in new specs and examples

* add new property and update fields to be required

* fix documentation link

* rename parameter

* fixed missed properties

* update golocal param

* update property

dhu5432 committed:
Add new API version for Microsoft.Resources/changes (2023-03-01-preview) (#23272)

* Base Commit. 2022-05-01 version for changes

* Changes for new api version 2023-03-01-preview

* Model as enum

* Forgot to include type

* isTruncated to string

* Change isTruncated to bool

jeffmcco-msft committed:
Change the default response schema to match actual behavior (#23363)

harshpatel17 committed:
Stacks/2022 08 01 preview (#23142)

* Added 2022-08 API Version files

* Added request changes

* Missed some file

* changed to v3 types

* Removed all snapshot references

---------

Co-authored-by: Harsh Patel <harshpatel@microsoft.com>

rajaharish committed:
added quota request example for Azure Lab Services (#23328)

dandandanliu915 committed:
[AutoSync] 30108fc4c3 Fixed Adls swagger to support expression (#23396)

Co-authored-by: swagger-automation <swagger@microsoft.com>

kazrael2119 committed:
credscan fix in swagger (#23400)

* credscan fix in swagger

* update files

jimmyca15 committed:
Update minimum snapshot retention period. (#23309)

srnagar committed:
Create stable version of log analytics query swagger (#23055)

* Create stable version of log analytics query swagger

* stable version

* fix linting issues

* fix readme

* fix lint issues

* add tags

* add default tag

pratyush-prateek committed:
Fixes for 2023-04-01-preview API version (#23427)

weshaggard committed:
Convert sample project to TypeSpec (#23168)

ramoka178 committed:
MGRP S360 Vuln (#22832)

* Add blockchain to latest profile

* Add additional types

* Fix Swagger issues

* Solve validation

---------

Co-authored-by: Mark Cowlishaw <markcowl@microsoft.com>

stuti-1807 committed:
Azure Orbital - swagger fix for api-version 2022-11-01 (#23441)

* added example for the new field

* updated the double ref that was causing build failure

giladsu committed:
Fixed PrometheusRuleGroups examples (#23390)

* Fixed PrometheusRuleGroups examples

* One more fix

koyasu221b committed:
Remvoe flattern (#23460)

Co-authored-by: Will Huang <huangwill@microsoft.com>

Commit:
Mvad update (#23434)

* Add default value 10 for topContributorCount

* Update AnomalyDetector typespec to latest typespec and Azure.Core versions and fix all warnings

* Update TypeSpec config

* Add back language emitter options

* Fix cspell and model validation errors

---------

Co-authored-by: Chunlei Wang <chuwan@microsoft.com>

namanag16 committed:
add codeowners for Compute Instance swagger (#23437)

Co-authored-by: Naman Agarwal <naagarw@microsoft.com>

davidgamero committed:
[Hub Generated] Review request for Microsoft.DevHub to add version preview/2022-10-11-preview (#22828)

* Adds base for updating Microsoft.DevHub from version preview/2022-04-01-preview to version 2022-10-11-preview

* Updates readme

* Updates API version in new specs and examples

* start 10-11 preview

* add words

* fix readme version

* update swagger version

* add second putworkflow example

* fix generatepreviewartifactsresponse

* align generate preview artifacts example

* update param locations that got changed

* add x-ms-client-flatten for artifact properties

* Adding WorkflowRunStatus

* Fixing enum name

* add namespace to example

---------

Co-authored-by: Brandon Foley <brandonfoley13@gmail.com>

msyyc committed:
Update readme.python.md (#23208)

karishmadaga committed:
fixing async response type for machinelearningservices-2023-02-01-preview (#23105)

* fixing regex pattern and async response type

* remove update to regex

vikotha committed:
Fix lint error for Datadog RP (#23477)

* Fix link error for Datadog RP

* Fix version

banggaurav committed:
merging billing fix to public repo (#23424)

Co-authored-by: Gaurav Bang <gauravbang@microsoft.com>

Wzb123456789 committed:
Update readme.python.md (#23478)

* Update readme.python.md

* Update readme.python.md

* update

* Update readme.python.md

* fix

* Update readme.python.md

---------

Co-authored-by: Yuchao Yan <yuchaoyan@microsoft.com>
Co-authored-by: Zhenbiao Wei (WICRESOFT NORTH AMERICA LTD) <v-zhenbwei@microsoft.com>

Vibhuti-Sharma-Microsoft committed:
updated specs (#23435)

RemindD committed:
Fix api-version in examples (#23342)

Co-authored-by: Xingdong Li <xingdli@microsoft.com>

priyankarking committed:
Add 202 status code for PrivateEndpointConnections_Delete (#23403)

Co-authored-by: Priyankar Gupta <priyagupta@microsoft.com>

priyankarking committed:
Add for 202 status code for PrivateEndpointConnections_Delete (#23401)

Co-authored-by: Priyankar Gupta <priyagupta@microsoft.com>

minhanh-phan committed:
Update Codeowners for Language/ SchemaRegistry (#23467)

purbch committed:
Update gallery.json

purbch committed:
Merge branch 'Azure:main' into purbch/api/validations-profile-swagger
